wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+  <v-progress-linear v-if="isLoading" color="primary" indeterminate />
+  <v-card class="ma-10">
+    <v-row align="center">
+      <v-col cols="6" align="center">
+        <v-tooltip
+          text="You must be Master of the Universe and 21 or older to add words"
+          :disabled="isOlderThanTwentyOne && isMotU">
+          <template v-slot:activator="{ props }">
+            <v-btn
+              v-bind="props"
+              :variant="isOlderThanTwentyOne && isMotU ? 'elevated' : 'plain'"
+              :color="isOlderThanTwentyOne && isMotU ? 'primary' : ''"
+              class="ma-0"
+              @click="if (isOlderThanTwentyOne && isMotU) addWordDialog = true;"
+              >Add word</v-btn
+            >
+          </template>
+        </v-tooltip>
+      </v-col>
+      <v-col>
+        <v-radio-group v-model="commonRadio">
+          <v-radio
+            label="Both common/uncommon"
+            :value="CommonRadio.Both"></v-radio>
+          <v-radio label="Only common" :value="CommonRadio.Common"></v-radio>
+          <v-radio
+            label="Only uncommon"
+            :value="CommonRadio.Uncommon"></v-radio>
+        </v-radio-group>
+      </v-col>
+    </v-row>
+    <v-table class="table my-2 mx-auto w-75 opacity-50">

```

Comment:
You seem to be recreating a lot of the functionality of the [Vuetiy's V Data Table](https://vuetifyjs.com/en/components/data-tables/basics/#usage) manually such as sorting. I would suggest looking into this as it will give you a lot of this stuff right out the box with mimical configuration.

Reply:
Thank you will look at it

Reply:
That's a thing?! Wow. That makes all of this so much simpler. Thank you for pointing that out!